client/my-sites/email/email-providers-comparison/in-depth/data.tsx

```diff
@@ -3,12 +3,19 @@
 import { Gridicon } from '@automattic/components';
 import { translate } from 'i18n-calypso';
 import googleWorkspaceIcon from 'calypso/assets/images/email-providers/google-workspace/icon.svg';
+import poweredByTitanLogo from 'calypso/assets/images/email-providers/titan/powered-by-titan-caps.svg';
 import { getGoogleMailServiceFamily } from 'calypso/lib/gsuite';
 import { getTitanProductName } from 'calypso/lib/titan';
 import { ADDING_GSUITE_TO_YOUR_SITE, ADDING_TITAN_TO_YOUR_SITE } from 'calypso/lib/url/support';
 import type { EmailProviderFeatures } from 'calypso/my-sites/email/email-providers-comparison/in-depth/types';
 
 export const professionalEmailFeatures: EmailProviderFeatures = {
+	badge: (
+		<img
+			alt={ translate( 'Powered by Titan icon', { textOnly: true } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Powered by Titan' )` *ES Score: 8*